.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "protoc-gen-prost"]
+	path = protoc-gen-prost
+	url = https://github.com/SuperFluffy/protoc-gen-prost
+	branch = bump-tonic
```

Comment:
We should probably target a commit sha so that updates to the branch don't break our pipelines.